ansible/roles/bie-hub/templates/bie-hub-config.properties

```diff
@@ -0,0 +1,33 @@
+bie.index.url={{ bie_index_url | default('http://localhost:8090/bie-index') }}
```

Comment:
Im not sure what sensible default is. Ive changed to point to dev environment for now (in the absence of an ALA prod environment).